### DISCUSSION TITLE
Should I use the Core TCTL/TDIE Or Core TDIE?

### DISCUSSION THREAD

**Discussion by PrinceLikesCookies** [Rem0o/FanControl.Releases#534] [Q&A] [answered]
I know the core tctl/tdie has a degree offset but I don't know if I should which since core tdie has lower temperatures and is the real temperature of the cpu. I don't know the correct one to cool down.

**Comment by Rem0o** [comment]
It doesn't really matter. They have a different range, but both will scale pretty much the same way under load. So simply adjust your curve to either and you'll be fine.

**Comment by AtA3301** [comment, marked as answer]
> I know the core tctl/tdie has a degree offset but I don't know if I should which since core tdie has lower temperatures and is the real temperature of the cpu. I don't know the correct one to cool down.

If you still can't decide, maybe make a "Mix Custom Sensor" in Fan Control, and set it to "Max" or "Average". 😀
